cirq_superstaq/compiler_output.py

```diff
@@ -32,7 +32,7 @@ def __init__(
         pulse_sequences: Optional[Any] = None,
         seq: Optional["qtrl.sequencer.Sequence"] = None,
         jaqal_programs: Optional[Union[List[str], str]] = None,
-        pulse_lists: Optional[Union[List[List], List[List[List]]]] = None,
+        pulse_lists: Optional[Union[List[List[str]], List[List[List[str]]]]] = None,
```

Comment:
```suggestion
        pulse_lists: Optional[Union[List[List[List[Any]]], List[List[List[List[Any]]]]]] = None,
```

https://github.com/SupertechLabs/qiskit-superstaq/pull/234#discussion_r1006241901